uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/services/settings.js

```diff
@@ -78,4 +89,29 @@ define([
 
     }]);
 
+    services.factory('AppSettingsService',['AppSettings', '$q', 'config', 'UtilsService', '$log',
+        function (AppSettings, $q, config, UtilsService, $log) {
+        $log.debug('Service: SettingsService');
+
+        return {
+            get: function(fields){
```

Comment:
Please provide a documentation to the function

Reply:
will do.